tests/test_urls.py

```diff
@@ -147,6 +150,13 @@ def setUp(self):
         self.item_b = NewsItem.objects.create(
             newsindex=self.index_b, title='Post B', date=dt(2015, 8, 2))
 
+    @classmethod
+    def tearDownClass(cls):
+        super(TestMultipleSites, cls).tearDownClass()
+        # Clear site cache when the tests finish to prevent other tests being
+        # polluted by a stale cache.
+        cache.delete('wagtail_site_root_paths')
```

Comment:
It is set by Wagtail: https://github.com/wagtail/wagtail/blob/v1.9/wagtail/wagtailcore/models.py#L201-L216